### ISSUE TITLE
Can't find the plugin

### ISSUE THREAD

**Issue by turpez** [fatheroctopus/hdt-deck-predictor#9]
The plugin is not displayed in the plugins on the tracker

**Comment by fatheroctopus** [comment]
Yep, longstanding issue, but I've abandoned this project so someone else will need to submit a fix. I took someone else's fix previously and rebuilt and re-released, but people said it didn't work.

**Comment by theanine** [comment]
https://stackoverflow.com/questions/1091853
